docs/contributor-guide/developer-guide/deploy-system/prerequisits.md

```diff
@@ -3,12 +3,256 @@
+
+4. Verify that you are now viewing the newly created account by navigating to the [IAM dashboard](https://us-east-1.console.aws.amazon.com/iamv2/home?region=us-east-1#/home) and making sure that the Account ID listed in the sidebar on the right matches the account ID for your newly created dev account.
+![Check new dev account number](../../../.gitbook/assets/aws-confirm-switched-roles.png)
+
+Note: You only have to log in to your dev account one time. After that, it will be listed under "Role history" in the drop down in the upper right and you will be able to easily toggle back and forth between accounts by clicking on the role in that menu.
+
+![Role history in dropdown](../../../.gitbook/assets/aws-role-history.png)
+
+### Configure new AWS Account
+
+1. Navigate to your newly created dev account's [IAM page](https://us-east-1.console.aws.amazon.com/iamv2/home?region=us-east-1#/home).
+
+2. Click the "Create" button next to the Account Number on the righthand sidebar.
+![Create Alias button](../../../.gitbook/assets/aws-create-account-alias.png)
+
+3. Under "Preferred alias" input the name for your newly created dev account (eg. `jdoe-dev`) and click "Save changes". You will use this alias later when running `aws-nuke` to clear resources.
+![Create Alias screen](../../../.gitbook/assets/aws-enter-preferred-alias.png)
+
+4. Navigate to the [Request certificate page](https://us-east-1.console.aws.amazon.com/acm/home?region=us-east-1#/certificates/request). Keep the default "Request a public certificate" option selected and click the "Next" button.
+![Request public certificate page](../../../.gitbook/assets/aws-request-public-cert.png)
+
+5. Input the certificate details and click "Request":
+    - Fully qualified domain name: Input `<your-dev-account-name>.civiform.dev` (eg. `jdoe-dev.civiform.dev`)
+    - Validation method: Leave the default selection "DNS validation - recommended" selected.
+    - Key algorithm: Leave the default selection "RSA 2048" selected.
+![Enter certificate details page](../../../.gitbook/assets/aws-enter-cert-details-page.png)
+
+6. You should be redirected to the [Certificates page](https://us-east-1.console.aws.amazon.com/acm/home?region=us-east-1#/certificates/list) and there should be a banner at the top of the page that says "Successfully requested certificate with ID ...". Click "View certificate". This will take you to the certificate details page.
+![View certificate banner](../../../.gitbook/assets/aws-view-cert-banner.png)
+
+7. In the "Domains" section of the certificate details page, there should be a single row with `CNAME` listed under the "Type" column. The values in the "CNAME name" and "CNAME value" columns are used to validate that we control the domain (`civiform.dev`) requested by the certificate. Copy both the "CNAME name" and "CNAME value" column values somewhere for yourself temporarily, you will need them in the following steps.
```

Comment:
For some reason, this added a `.` at the end of both strings when clicking the "copy" buttons on the page, which causes the Google Domains input to complain.